build.sbt

```diff
@@ -255,6 +255,10 @@ lazy val coreTests = crossProject(JSPlatform, JVMPlatform, NativePlatform)
   .dependsOn(testRunner)
   .settings(buildInfoSettings("zio"))
   .settings(publish / skip := true)
+  .settings(
```

Comment:
You can use the `typecheck` operator in ZIO Test for this. There is no need for a third party dependency.

Reply:
Done